src/main/java/racingcar/domain/round/Round.java

```diff
@@ -0,0 +1,46 @@
+package racingcar.domain.round;
+
+public class Round {
+
+    private static final int MIN_NUMBER = 1;
+
+    private int round;
+
+    public Round(String round) {
+        this(parseInt(round));
+    }
+
+    public Round(int round) {
+        validate(round);
+        this.round = round;
+    }
+
+    private void validate(int round) {
+        if (round < MIN_NUMBER) {
+            throw new IllegalArgumentException("1 이상의 양수를 입력하세요.");
+        }
+    }
+
+    private static int parseInt(String round) {
+        try {
+            return Integer.parseInt(round);
+        } catch (NumberFormatException e) {
+            throw new IllegalArgumentException("1 이상의 양수를 입력하세요.");
+        }
+    }
+
+    public boolean isEnd(int endRound) {
+        if (endRound > round) {
+            throw new IllegalArgumentException("현재 시도 회수가 끝나는 시도 회수보다 작을 수 없습니다.");
+        }
+        return this.isSame(endRound);
+    }
+
```

Comment:
시도 횟수가 0이면 더 이상 시도할 수 없는 거나 마찬가지 아닌가요?
endRound가 바뀔 수 있나요?

```suggestion
    public boolean isEnd() {
        return round == END_OF_ROUND;
    }
```